src/main/scala/net/psforever/actors/session/support/ZoningOperations.scala

```diff
@@ -1442,8 +1442,22 @@ class ZoningOperations(
     if (player.avatar.vehicle.nonEmpty && player.VehicleSeated != player.avatar.vehicle) {
       continent.GUID(player.avatar.vehicle) match {
         case Some(vehicle: Vehicle) if vehicle.Actor != Default.Actor =>
-          sessionData.vehicles.TotalDriverVehicleControl(vehicle)
+
+          // allow AMS, router, ANT and BFR siphon to remain deployed when owner leaves the zone
+          // TODO: do we need to check for left/right siphon?
+          vehicle.Definition match {
+            case GlobalDefinitions.ams | GlobalDefinitions.ant | GlobalDefinitions.router |
+                 GlobalDefinitions.aphelion_ntu_siphon | //GlobalDefinitions.aphelion_ntu_siphon_left | GlobalDefinitions.aphelion_ntu_siphon_right |
```

Comment:
`GlobalDefinitions.aphelion_ntu_siphon` is the arm weapon and should never resolve to the vehicle.

Perform `sessionData.shooting.findContainedWeapon` and then look for the ammo type `Ammo.ntu_siphon_ammo` on any output tool (`tool.AmmoType`).  It's the second set of values in the output pair.

Additionally, or alternately, please switch to `ConditionalDriverVehicleControl`.  I wish to avoid losing driver control of any vehicle if it is moved by the server (as it may) into a weird position.

Reply:
BFR siphon does not function like a deployment and is not continous. It requires the player to press the fire button and as such does not need to be added to this logic.
I removed them from the check.